internal/policy/root.go

```diff
@@ -146,6 +149,58 @@ func DeleteTargetsKey(rootMetadata *tuf.RootMetadata, keyID string) (*tuf.RootMe
 	return rootMetadata, nil
 }
 
+// AddGitHubAppKey adds the 'appKey' as a trusted public key in 'rootMetadata'
+// for the special GitHub app role.
+func AddGitHubAppKey(rootMetadata *tuf.RootMetadata, appKey *tuf.Key) (*tuf.RootMetadata, error) {
```

Comment:
Is there anything special about this key that's specific to GitHub Apps, or is this a generic "Attestor" role?

Reply:
In particular, I want to avoid confusion with https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/managing-private-keys-for-github-apps

Reply:
From the gittuf side, this is special. During the verification flow, we trust the contents of the github app predicate (which says X reviewed this change) only if it's signed by this key specified in the root.

FWIW, (I'll double check), I'm currently reusing the key used by the app to communicate with github to also sign the attestation the app generates, so it's currently the same key. We may want to change this, though, if the app uses an identity + sigstore for example.

https://github.com/gittuf/github-app/pull/1/files#diff-59d92a25c9a864d2527aa8c12116ba798ab65673cf86cd0759b4c1075d01653bR299

Reply:
fixed docs for this